README.md

```diff
@@ -104,6 +135,8 @@ With everything set up, the benchmark should be ready to run! Here's a last-minu
 - Close all background apps
 - Close all open tabs in the browsers that will be tested
 - Close out of said browsers and force stop them
+- Run a sanity check to verify the browser your results are sane
+- Charge to at above 30% (some OEMs are known to cap the performance when the phone is running out of charge)
```

Comment:
Consider putting as the first step of the preflight checklist (no point in doing all the other prep if you have to charge it anyway).

Also, suggested wording:

```suggestion
- Ensure the device is fully charged and Battery Saver is not on (some OEMs are known to cap the performance when the phone is running out of charge or on Battery Saver)
```